go.mod

```diff
@@ -83,6 +83,7 @@ require (
 	github.com/containerd/stargz-snapshotter/estargz v0.14.3 // indirect
 	github.com/cyphar/filepath-securejoin v0.2.4 // indirect
 	github.com/evilmonkeyinc/jsonpath v0.8.1 // indirect
+	github.com/expr-lang/expr v1.16.0 // indirect
```

Comment:
You'll want to change the previous [individual reference](https://github.com/argoproj/argo-workflows/blob/8d27a9f12a78f6a81998023044872ec1006e2f78/go.mod#L15) under `antonmedv` to be under the new org location `expr-lang` instead. The imports may need changing as well